src/button/button.directive.ts

```diff
@@ -8,6 +8,8 @@ import {
 /**
  * A convinence directive for applying styling to a button.
  *
+ * demo: [https://angular.carbondesignsystem.com/?path=/story/button--basic](../../?path=/story/button--basic)
```

Comment:
Not sure we need a minor version change for this since it doesn't add any additional functionality for the dev. Changes are to docs and demos. nbd, food for thought

Reply:
Yep, this should just be a `chore(docs):`

Reply:
We added a doc's component and did some changes to the welcome story

Reply:
But none of the actual changes to components, i.e. when a dev pulls a new version, what additional features do they have? What bug fixes do they have? Will anything break?